### PR TITLE
Restore "CBM:Solar Panels" in Bright Nights mod

### PR DESCRIPTION
**Pull request by John-Candlebury**
Moved to a mod, as Kevin suggested in #23317

### CODE REVIEW

File: data/mods/BrightNights/item_groups.json [hunk 1]
```diff
@@ -6,6 +6,51 @@
...
+    "id" : "bionics_op",
+    "type": "item_group",
+    "magazine": 100,
+    "ammo": 50,
+    "items": [
+      [ "bio_solar", 15 ]
+    ]
+  },
+  {
+    "id" : "bionics_op",
```
Comment (Night-Pryanik) [inline data/mods/BrightNights/item_groups.json]: This item_group is identical to the item_group above.

**Comment by Night-Pryanik** [comment]
You can also add solar panels as default bionics to the Bionic Sniper profession, as it was before their removal.

File: data/mods/BrightNights/bionics.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "id": "bio_solar",
+    "type": "bionic",
+    "name": "Solar Panels",
+    "description": "Installed on your back is a set of retractable solar panels.  When in direct sunlight, they will automatically deploy and slowly recharge your power level.",
```
Comment (Night-Pryanik) [inline data/mods/BrightNights/bionics.json]: You decided to make them automatically deploy, not retract and fold by will?

Comment (John-Candlebury) [reply data/mods/BrightNights/bionics.json]: Requires less atention from the player, and also had waves why they wont get damaged when fighting. Seems convenient all around.

**Comment by John-Candlebury** [comment]
What even is Error 1?  Do I need add a linter rule somewhere?

**Comment by John-Candlebury** [comment]
>You can also add solar panels as default bionics to the Bionic Sniper profession, as it was before their removal.

I want to avoid making bright nights necessarily big, and scince the solar panels are not really essential to the play stile of the bionic sniper I rather not duplicate the profession.

**Comment by smolbird** [comment]
This should be in mainline to begin with.

**Comment by kevingranade** [comment]
You might want to page @Coolthulhu since it's his mod.

**Comment by Wishbringer** [comment]
How about adding something like a Radioisotope thermoelectric generator (RTG) to replace solar panels.

**Comment by John-Candlebury** [comment]
>  You might want to page @Coolthulhu since it's his mod.

The Pr already has his thumbs up, so I think it should be alright from that front.

--

>How about adding a Radioisotope thermoelectric generator (RTG) to remove the void left by the solar panels removal? I can see some interesting situations with radiation leakages and possible secondary effect as a heat source. :) Though generally they are quite safe and has been used to power heart pacemakers in the past.

The plutonium reactor bionic that is already in game is basically this.

**Pull request merged**